sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs

```diff
@@ -2185,26 +2185,15 @@ public virtual Response PublishTelemetry(string digitalTwinId, string messageId,
         /// <summary>
         /// Gets the scope for authentication/authorization policy.
         /// </summary>
-        /// <param name="endpoint">Azure digital twins instance Uri.</param>
         /// <returns>List of scopes for the specified endpoint.</returns>
-        internal static string[] GetAuthorizationScopes(Uri endpoint)
+        internal static string[] GetAuthorizationScopes()
         {
-            Argument.AssertNotNull(endpoint, nameof(endpoint));
-            Argument.AssertNotNullOrEmpty(endpoint.AbsoluteUri, nameof(endpoint.AbsoluteUri));
-
-            // Uri representation for azure digital twin app Id "0b07f429-9f4b-4714-9392-cc5e8e80c8b0" in the public cloud.
-            const string adtPublicCloudAppId = "https://digitaltwins.azure.net";
+            // Vanity representation for azure digital twin app Id "0b07f429-9f4b-4714-9392-cc5e8e80c8b0" in the public cloud
+            // and shared by other clouds.
+            const string adtDefaultCloudScope = "https://digitaltwins.azure.net";
             const string defaultPermissionConsent = "/.default";
 
-            // If the endpoint is in Azure public cloud, the suffix will have "azure.net" or "ppe.net".
-            // Once ADT becomes available in other clouds, their corresponding scope has to be matched and set.
-            if (endpoint.AbsoluteUri.IndexOf("azure.net", StringComparison.OrdinalIgnoreCase) > 0
-                || endpoint.AbsoluteUri.IndexOf("ppe.net", StringComparison.OrdinalIgnoreCase) > 0)
-            {
-                return new[] { adtPublicCloudAppId + defaultPermissionConsent };
-            }
-
-            throw new InvalidOperationException($"Azure digital twins instance endpoint '{endpoint.AbsoluteUri}' is not valid.");
+            return new[] { adtDefaultCloudScope + defaultPermissionConsent };
```

Comment:
nit: if this is really (probably) never going to change, we can just precompute it rather than concat'ing the string and allocating the array at runtime.